### PR TITLE
Use Wise Old Man API v2

### PR DESCRIPTION
**Pull request by rorro**

### CODE REVIEW

File: src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java [hunk 1]
```diff
@@ -557,16 +558,16 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 			return;
 		}
 
-		String name = Text.toJagexName(Text.removeTags(event.getTarget()).toLowerCase());
+		String name = Text.toJagexName(Text.removeTags(event.getTarget()));
```
Comment (dekvall) [inline src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java:561]: Why did you change this?

File: src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java [hunk 2]
```diff
@@ -958,8 +959,7 @@ public void onWomGroupSynced(WomGroupSynced event)
 	@Subscribe
 	public void onWomGroupMemberAdded(WomGroupMemberAdded event)
 	{
-		MemberInfo member = event.getMember();
-		groupMembers.put(member.getUsername().toLowerCase(), member);
+		womClient.importGroupMembers();
```
Comment (dekvall) [inline src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java:962]: Is this really required?

File: src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java (hunk 1 above)
Comment (rorro) [reply src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java:561]: Removing and adding players no longer returns the information about the added and removed players. Before this we got the display name from the response which was correctly capitalized and showed it in the message in-game. The new response doesn't contain any player info. So I removed the toLowerCase() so the message in-game can still show correctly capitalized username. It's used in `womClient.removeGroupMember(name)` and `womClient.addGroupMember(name)`.

The new response looks like this now.
```
{
    "count": 1,
    "message": "Successfully added 1 members."
}
```

File: src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java (hunk 2 above)
Comment (rorro) [reply src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java:962]: This has to be done for the same reason as the one above.
